include/boost/url/grammar/ascii.hpp

```diff
@@ -40,8 +41,35 @@ constexpr
 char
 ascii_tolower(char c) noexcept
```

Comment:
Can this be constexpr? Should it?

Reply:
I replicated the pattern from `ascii_tolower` but didn't think much about it before. I don't see why they shouldn't be `constexpr`.

Reply:
Oh it was already constexpr, but github diff just didn't show it..